### PR TITLE
Add a slide that introduces how to contribute to SQLFlow Models

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: cmd/demo/slides/20190620/sqlflow.slide [hunk 1]
```diff
@@ -0,0 +1,122 @@
+Contribute to SQLFlow
```
Comment (tonyyang-svail) [inline cmd/demo/slides/20190620/sqlflow.slide]: Maybe change to "Contribute to SQLFlow Models"?

File: cmd/demo/slides/20190620/sqlflow.slide [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+yancey.yx@antfin.com
+https://sqlflow.org
+
+* SQLFlow Open Source Community:
+- https://github.com/sql-machine-learning/
+  - 2800+ starts
+  - 17+ contributors
+  - 5 repos
+
+.image ./sqlflow-community.png 340 _
```
Comment (tonyyang-svail) [inline cmd/demo/slides/20190620/sqlflow.slide]: How about 
- "SQL Engine communication" => "Database community"
- "ML Engine community" => "ML community"

File: cmd/demo/slides/20190620/sqlflow.slide [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+18 June 2019
+
+Yan Xu
+Software Engineer, Ant Financial
+yancey.yx@antfin.com
+https://sqlflow.org
+
+* SQLFlow Open Source Community:
+- https://github.com/sql-machine-learning/
+  - 2800+ starts
```
Comment (tonyyang-svail) [inline cmd/demo/slides/20190620/sqlflow.slide]: Looks like `go present` doesn't support nested bullet point.
Screen Shot 2019-06-19 at 3 17 45 PM

File: cmd/demo/slides/20190620/sqlflow.slide [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+* The Architecture
+
+- Users input SQL statements in Jupyter Notebook
+- The SQLFlow magic command sends each statement to SQLFlow server
+- SQLFlow server proxies statements without using extended syntax to an SQL engine
+- SQLFlow server translates each extended-syntax statement into a (Python) submitter program and runs it.
+- The submitter program runs SELECT on SQL engine and reads results for training/prediction.
+- The submitter program TRAIN the MODEL in sqlflow-models.
+
+* Challenges
```
Comment (tonyyang-svail) [inline cmd/demo/slides/20190620/sqlflow.slide]: Not sure we need to present the challenge in an introduction oriented slide.

File: cmd/demo/slides/20190620/sqlflow.slide [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+* Design: Feature Derivation
+
+When we need to derive a new feature from combining multiple features, we use crossed feature:
+
+.code ../201906/train_cross.sql
+
+And this will generate feature column code like:
+
+.code ../201906/feature_column_cross.pysample
+
```
Comment (tonyyang-svail) [inline cmd/demo/slides/20190620/sqlflow.slide:92]: Maybe move the second slide `* SQLFlow Open Source Community` here. So we can present our git workflow right after the introduction of the open source community.

File: cmd/demo/slides/20190620/sqlflow.slide (hunk 4 above)
Comment (Yancey0623) [reply cmd/demo/slides/20190620/sqlflow.slide]: The *feature column* is related to the models, maybe users also would like to know how to support the certain feature type or how to custom the feature type, we can also listen to our users, how about changing the title to `Feature Column`

File: cmd/demo/slides/20190620/sqlflow.slide (hunk 5 above)
Comment (Yancey0623) [reply cmd/demo/slides/20190620/sqlflow.slide:92]: done.

File: cmd/demo/slides/20190620/sqlflow.slide (hunk 1 above)
Comment (Yancey0623) [reply cmd/demo/slides/20190620/sqlflow.slide]: done.

File: cmd/demo/slides/20190620/sqlflow.slide (hunk 2 above)
Comment (Yancey0623) [reply cmd/demo/slides/20190620/sqlflow.slide]: Maybe  "ML community" is too wide? Seems we only consider the ML System/Engine here.

File: cmd/demo/slides/20190620/clone_sqlflow_models.bashsample [hunk 6]
```diff
@@ -0,0 +1 @@
+git clone git@github.com:<your Github account>/models.git
```
Comment (typhoonzero) [inline cmd/demo/slides/20190620/clone_sqlflow_models.bashsample:1]: How can use one line code directly in `.slide` file?

**Review by typhoonzero — APPROVED** [review]

**Pull request merged**